### DISCUSSION TITLE
GitHub Student Account Not Getting Approved

### DISCUSSION THREAD

**Discussion by DevoidSloth** [community/community#78948] [GitHub Education]
I have a student email.
I need to verify my email but it is not recieving the verification message.
I have tried a bunch of other peoples solutions but none have worked.
What do I do about this? How can I verify my email?

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @DevoidSloth , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!

**Reply by DevoidSloth** [reply]
Thanks, sorry I put it here because I saw some other people did the same.

**Reply by queenofcorgis** [reply]
All good - just want to make sure you're getting the proper eyes on this 😃

**Comment by AidenJamesYt** [comment]
Troubleshooting the Verification Email:

Check your spam and junk folders: The verification email might be mistakenly filtered.
Add GitHub to your email's safe senders list: This ensures emails from GitHub don't get blocked.
Request a new verification email: Click the "resend verification" link on the GitHub Student Pack application page. You might need to log in again.
Use a different email address: If you have another student email address, try using that for verification.
Clear your browser cache and cookies: Sometimes cached data can interfere with email delivery.
If you are using a school email, make sure GitHub has message send permissions: Some schools/universities have email blockers from unknown handles.

Alternative Verification Methods:

Verify your academic affiliation directly: If your institution is listed on the GitHub Education Partners page, you can verify your affiliation directly through their system. This bypasses the email verification step. https://education.github.com/pack/partners
Contact GitHub Support: Explain your situation and the verification issues you're facing. They might be able to offer further assistance or troubleshoot the problem on their end. https://support.github.com/

Additional Tips:

Make sure you're using a valid student email address from an accredited institution.
Double-check your application details for any typos or errors.
Review the GitHub Student Pack eligibility requirements to ensure you meet all the criteria. https://education.github.com/pack
If you've tried all of these options and still haven't received the verification email or your application isn't getting approved, contacting GitHub Support is the best course of action. They'll be able to investigate the issue further and provide personalized assistance. [Here is how you can open a support ticket](https://docs.github.com/en/support/contacting-github-support)

**Reply by queenofcorgis** [reply]
Hi @AidenJamesYt , 

We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. Thank you for helping us maintain an authentic and beneficial space for everyone.
